### PR TITLE
[Calling][Bugfix] Skip setup and multitasking

### PR DESCRIPTION
**Pull request by pavelprystinka**
## Purpose

* Skip setup and multitasking. When skip setup and multitasking with Pip ware enabled, call composite did close, rather then show up in Pip
* Hide lobby admit notification while in the Pip

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

## How to Test

* Open...
* Click on...

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### CODE REVIEW

File: AzureCommunicationUI/Podfile.lock [hunk 1]
```diff
@@ -84,4 +84,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 78ea59a451b84782ef840c80dc01e9afa5adf125
```
Comment (jamchengms) [inline AzureCommunicationUI/Podfile.lock:86]: can we revert this file if it's not needed?
would prefer we go up in version if we are changing the cocoapods versioning

**Review by jamchengms — APPROVED** [review]

File: AzureCommunicationUI/Podfile.lock (hunk 1 above)
Comment (pavelprystinka) [reply AzureCommunicationUI/Podfile.lock:86]: I will update my tools.

**Review by jamchengms — APPROVED** [review]

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/SwiftUI/Setup/SetupViewModel.swift [hunk 2]
```diff
@@ -105,12 +105,6 @@ class SetupViewModel: ObservableObject {
         networkManager.stopMonitor()
     }
 
-    func dismissSetupScreen() {
-        if store.state.callingState.operationStatus == .skipSetupRequested {
```
Comment (Mohtasim) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/SwiftUI/Setup/SetupViewModel.swift:109]: How are we skipping setup screen now?

**Review by Mohtasim — APPROVED** [review]

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/SwiftUI/Setup/SetupViewModel.swift (hunk 2 above)
Comment (pavelprystinka) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/SwiftUI/Setup/SetupViewModel.swift:109]: in the CallingMiddlewareHandler.setupCall()

**Pull request merged**